tf_agents/agents/dqn/dqn_agent.py

```diff
@@ -14,9 +14,7 @@
 # limitations under the License.
 
 """A DQN Agent.
-
```

Comment:
Undo all the space removal.

Reply:
You were right, several spaces disappeared. Thank you.